WordPress/WordPressTest/ContextManagerTests.swift

```diff
@@ -155,6 +155,19 @@ class ContextManagerTests: XCTestCase {
         }
         XCTAssertEqual(numberOfAccounts(), 1)
 
+        do {
+            try await contextManager.performAndSave { context in
+                let account = WPAccount(context: context)
+                account.userID = 100
+                account.username = "Unknown User"
+                throw NSError(domain: "save", code: 1)
+            }
+            XCTFail("The above call should throw")
```

Comment:
Nitpick: Would you consider a more verbose version of the failure message?

```suggestion
            XCTFail("Expected `performAndSave` to throw error, but this code was reached so no error was thrown.")
```